sdk/loadtestservice/Azure.ResourceManager.LoadTestService/src/autorest.md

```diff
@@ -0,0 +1,62 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: LoadTestService
+namespace: Azure.ResourceManager.LoadTestService
```

Comment:
Question: did the namespace got reviewed by archboard?

Reply:
@ArthurMa1978 We had followed the documentation as per: [.NET SDK release](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=build)
The documentation doesn't callout the arch-board review for SDK management plane.
Please let us know whom we can reach out to.

Reply:
Let's merge this PR first and open another separate PR if we need changes for the namespace in the future.